deployments/dev/config/staging.yaml

```diff
@@ -3,12 +3,12 @@ pangeo:
     proxy:
       https:
         hosts:
-        - staging.hub.pangeo.io
+          - staging.hub.pangeo.io
         letsencrypt:
           contactEmail: jhamman@ucar.edu
+    hub:
       service:
         loadBalancerIP: 35.232.186.70
-    hub:
```

Comment:
@jhamman I mean that this moving of the proxy.service.loadBalancerIP to hub.service.loadBlanacerIP will break things. The hub k8s service should not have the external IP of the deployment.

Reply:
Gotcha. So should I just remove the `loadBalancerIP` all together? Is this needed anymore?

Reply:
> This PR updates our proxy/service configs to match the current layout in zero-to-jupyterhub: https://zero-to-jupyterhub.readthedocs.io/en/latest/administrator/security.html#set-up-automatic-https

I'm not understood what you referred to so I'm very confused in general. But, as far as I know, there shouldn't be any change to the proxy.service.loadBalancerIP configuration due to an upgrade from 0.9.0-beta.2 to 0.9.0-beta.3 or 4, and I'm confident it should not be moved from proxy.service.loadBalancerIP to hub.service.loadBalancerIP

Reply:
the confusion on my end is that `loadBalancerIP` is not mentioned anywhere on the page linked above. Based on those docs, its not clear to me if `loadBalancerIP` is needed or where it should be applied.

Reply:
Ah I see!

It is used when a service is of `type: LoadBalancer`, then you can request a specific IP if that IP is available for use. That way you ensure you always have the same IP for the various deployments, which will be very relevant for domain names referencing fixed IPs.

So, to summarize, it is not essential for HTTPS, but it is a recommended practice to ensure you don't end up with a different IP if you reinstall this Helm chart or similar. I think it would make sense to have documentation or a mention about this in https://zero-to-jupyterhub.readthedocs.io/en/latest/administrator/security.html#set-up-automatic-https!

Reply:
Got it. I've rolled back all the loadBalancerIP changes.

Reply:
And yes! it would be great to tie these two pieces of documentation together.

Reply:
I opened this issue to represent that: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/1599